gtfs-validator/main/src/test/java/org/mobilitydata/gtfsvalidator/validator/TransfersAreUniqueValidatorTest.java

```diff
@@ -0,0 +1,119 @@
+                                              .build();
+  private final GtfsTransfer transfer_2_duplicate = new GtfsTransfer.Builder()
+                                                        .setFromStopId("stop_3")
+                                                        .setToStopId("stop_7")
+                                                        .setCsvRowNumber(4)
+                                                        .build();
+  private final GtfsTransfer transfer_3 = new GtfsTransfer.Builder()
+                                              .setFromStopId("stop_5")
+                                              .setToStopId("stop_9")
+                                              .setCsvRowNumber(3)
+                                              .build();
+
+  @Test
+  public void allDifferentTransfersShouldNotGenerateNotice() {
+    final NoticeContainer noticeContainer = new NoticeContainer();
+    List<GtfsTransfer> transfers =
+        new ArrayList<>(Arrays.asList(transfer_1, transfer_2, transfer_3));
+    validator.transferTable = GtfsTransferTableContainer.forEntities(transfers, noticeContainer);
+    validator.validate(noticeContainer);
+    assertThat(noticeContainer.getNotices()).isEmpty();
+  }
+
+  @Test
+  public void onlyDuplicateTransfersShouldGenerateNotice() {
+    final NoticeContainer noticeContainer = new NoticeContainer();
+    List<GtfsTransfer> transfers = new ArrayList<>(Arrays.asList(transfer_1, transfer_1_duplicate));
+    validator.transferTable = GtfsTransferTableContainer.forEntities(transfers, noticeContainer);
+    validator.validate(noticeContainer);
+    assertThat(noticeContainer.getNotices())
+        .containsExactly(new TransfersAreUniqueNotice(
+            /* fromStopId= */ "stop_1",
+            /* toStopId= */ "stop_2",
+            /* duplicateCsvRowNumber= */ 10,
+            /* originalCsvRowNumber= */ 1));
+  }
+
+  @Test
+  public void someDuplicateTransfersShouldGenerateNotice() {
+    final NoticeContainer noticeContainer = new NoticeContainer();
+    List<GtfsTransfer> transfers = new ArrayList<>(Arrays.asList(
+        transfer_1, transfer_2, transfer_3, transfer_1_duplicate, transfer_2_duplicate));
+    validator.transferTable = GtfsTransferTableContainer.forEntities(transfers, noticeContainer);
+    validator.validate(noticeContainer);
+    assertThat(noticeContainer.getNotices())
+        .containsExactly(new TransfersAreUniqueNotice(
+                             /* fromStopId= */ "stop_1",
+                             /* toStopId= */ "stop_2",
+                             /* duplicateCsvRowNumber= */ 10,
+                             /* originalCsvRowNumber= */ 1),
+            new TransfersAreUniqueNotice(
+                /* fromStopId= */ "stop_3",
+                /* fromStopId= */ "stop_7",
+                /* duplicateCsvRowNumber= */ 4,
+                /* originalCsvRowNumber= */ 2));
+  }
+
+  @Test
+  public void moreDuplicateTransfersShouldGenerateNotice() {
```

Comment:
The name is a bit confusing for me - as it actually represents one duplicate transfer pair among some transfers, while I cannot get it from "more duplicate transfers". Maybe have a better name?

Reply:
fixed! I changed the name to `anotherDuplicateTransferShouldGenerateNotice`. Name is trying to say that I have another test for a transfer. Not necessarily _another transfer_.